terminal/partials/page-content/revision.html

```diff
@@ -31,11 +31,14 @@
             {%- if config.repo_name == "Bitbucket" -%}
             See revision history on <a href="{{ page.edit_url | replace('mode=edit','mode=read') }}">{{ config.repo_name }}</a>.
             {%- endif -%}
+            {%- if config.repo_name == "Codeberg" -%}
+            See revision history on <a href="{{ page.edit_url | replace('_edit','commits/branch') }}">{{ config.repo_name }}</a>.
```

Comment:
hi @mark-pitblado i realized there is one edge case where if the name of the markdown file includes '_edited' then this will break

i think we can fix it by adding slashes though
```suggestion
            See revision history on <a href="{{ page.edit_url | replace('/_edit/','/commits/branch/') }}">{{ config.repo_name }}</a>.
```

before: `https://codeberg.org/ntno/demo/commits/branch/main/docs/draft_1commits/branched.md`
after: `https://codeberg.org/ntno/demo/commits/branch/main/docs/draft_1_edited.md`